h/api/models/document.py

```diff
@@ -38,16 +38,19 @@ class Document(Base, mixins.Timestamps):
                                backref='document',
                                order_by='DocumentMeta.updated.desc()')
 
+    meta_title = sa.orm.relationship('DocumentMeta',
+                                     primaryjoin='and_(Document.id==DocumentMeta.document_id, DocumentMeta.type==u"title")',
+                                     order_by='DocumentMeta.updated.asc()',
+                                     uselist=False,
+                                     viewonly=True)
+
```

Comment:
This produces warnings when there is more than one title document meta record (`SAWarning: Multiple rows returned with uselist=False for eagerly-loaded attribute 'Document.meta_title'`).
It does produce what we want, but I have the feeling that I'm using SQLAlchemy the wrong way to achieve what I want to achieve here. Any ideas?